flame_utils/io/lattice.py

```diff
@@ -81,17 +81,7 @@ def generate_latfile(machine, latfile=None, state=None, original=None, out=None)
         if isinstance(state, BeamState):
                 state = state._states
         if isinstance(state, flame._internal.State):
```

Comment:
Line 81-83 could be deleted since `generate_source` could handle both `BeamState` and `flame._internal.State` object.

Reply:
I've just updated the docstring of `generate_source` function.